Multisig/UI/Transaction/ExecuteTransaction/ReviewExecutionCellBuilder.swift

```diff
@@ -46,42 +48,98 @@ class ReviewExecutionCellBuilder: TransactionDetailCellBuilder {
         result.append(cell)
     }
 
+    func buildLabel(text: String) {
```

Comment:
This is the only place LabelCell is used, but this method is not called, if so please remove it with LabelCell files